src/app/modules/chart-module/components/common-chart/index.tsx

```diff
@@ -147,8 +147,9 @@ export function CommonChart(props: Props) {
         css={`
           width: 100%;
           overflow: hidden;
+
           margin-top: 40px;
-          height: ${props.visualOptions.height - (props.withHeader ? 36 : 0)}px;
+          height: ${props.visualOptions.height}px;
```

Comment:
did you check if this breaks any of the other chart types?

Reply:
yes I did. Doesn't break the other chart types